.test-infra/jenkins/job_PerformanceTests_SpannerIO_Python.groovy

```diff
@@ -93,7 +93,7 @@ def spannerio_read_test_2gb = [
     influx_db_name       : InfluxDBCredentialsHelper.InfluxDBDatabaseName,
     influx_hostname      : InfluxDBCredentialsHelper.InfluxDBHostUrl,
     input_options        : '\'{' +
-    '"num_records": 10485760,' +
+    '"num_records": 2097152,' +
```

Comment:
I am seeing that SpannerIO Read 10GB was previously failing, but this change modifies only 2gb stanzas.

Reply:
Yes, I've fixed the data size for 2GB.

The 10GB read was failing because of insufficient data size.

Read test works in a two-step. 
1: create (write pipeline) test dataset. Pass this step if the database already exists.
2: perform a read test.

Due to system timeout, (step-1) the write pipeline was canceled before finishing which results in insufficient data size and since we reuse the same data for the test it will always fail, so I have deleted the database manually in order to recreate it with the required size.

For the timeout issue, I increased it to 480 mins (8 Hours), but somehow job still canceling in around 4 hours which causes the current failures.
```
def executeJob = { scope, testConfig ->
  commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 480)
```